mantidproject/mantid
# Force slice viewer to stay on top of Mantid

_Must be tested on OSX_

This is a continuation of ticket #16817. The slice viewer is not a floating window (perhaps it should be!?) which provides the same fix with it being hidden behind the main window on OSX.

**To test:**
Before applying the change open the slice viewer then click the main mantid window. The slice viewer will disappear behind the main window. 

Apply the fix and the window should now stay in front of the main window even when focus is lost.

Fixes #16912.
## _Does not need to be in the release notes._
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
